--- statgrab.c	2012-05-06 11:23:14.751395802 +0200
+++ statgrab.c.oden	2012-05-06 11:23:31.253502922 +0200
@@ -69,7 +69,7 @@ static int le_statgrab;
  *
  * Every user visible function must have an entry in statgrab_functions[].
  */
-function_entry statgrab_functions[] = {
+zend_function_entry statgrab_functions[] = {
 	PHP_FE(sg_cpu_percent_usage,	NULL)
 	PHP_FE(sg_cpu_totals,		NULL)
 	PHP_FE(sg_cpu_diff,		NULL)
